test/system/commander/index.js

```diff
@@ -0,0 +1,43 @@
+const program = require('commander');
+
+let allowedConfig = ['ProdConfig', 'DevConfig'];
+let allowedScenarios = ['TestSingleTrader', 'TestTwoTrader', 'TestBalances'];
+
+const Commander = (arg) => {
+  program
+    .command('scenario <scenarioName>')
+    .alias('s')
+    .description('Choose Scenario To execute')
+    .action((name) => {
+      if (allowedScenarios.includes(name)) {
+        console.log('selected scenario is', name);
+        arg.value = name;
+        arg.option = 'scenario';
+      } else {
+        console.log('Scenario ', name, ' is not allowed. Allowed values are :', allowedScenarios.toString());
+        process.exit(1);
+      }
+    });
+
+  program
+    .command('config <config>')
+    .alias('c')
+    .description('Choose config')
+    .action((config) => {
+      if (allowedConfig.includes(config)) {
```

Comment:
The config option should specify the filename (not the config type).
To validate - we can use fs.existsSync(path)